tests/test_addon_detail.py

```diff
@@ -310,3 +310,64 @@ def test_more_info_external_license(selenium, base_url, variables):
     assert variables['base_url'] not in selenium.current_url
 
 
+@pytest.mark.nondestructive
+def test_more_info_custom_license(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    # checks that the AMO custom license page opens and has the correct content
+    custom_license = addon.more_info.addon_custom_license()
+    assert 'Custom License for Ghostery - Privacy Ad Blocker' in \
+           custom_license.custom_licence_and_privacy_header
+    assert custom_license.custom_licence_and_privacy_text.is_displayed()
+    assert custom_license.custom_licence_and_privacy_summary_card.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_more_info_privacy_policy(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    # enclosing in a try:except method as privacy policy is optional
+    # and not all add-ons will have it
+    try:
+        privacy = addon.more_info.addon_privacy_policy()
+        # checks that the AMO privacy policy page opens and has the correct content
+        assert 'Privacy policy for Ghostery - Privacy Ad Blocker' in \
+               privacy.custom_licence_and_privacy_header
+        assert privacy.custom_licence_and_privacy_text.is_displayed()
+        assert privacy.custom_licence_and_privacy_summary_card.is_displayed()
+    except NoSuchElementException:
+        print("The add-on doesn't have a privacy policy")
```

Comment:
If this test passes pytest may not print this line. I think this test should be split. One for testing an addon with a privacy policy and one for an addon without a policy

Reply:
I've added a separate test for add-ons without a Privacy Policy for better coverage. I did the same for the eula test.